src/main/java/pl/jakubowskir/timetable/trainer/controller/TrainerDto.java

```diff
@@ -0,0 +1,9 @@
+package pl.jakubowskir.timetable.trainer.controller;
+
+//import lombok.Getter;
+
+record TrainerDto {
```

Comment:
tak się nie tworzy rekordów, to powinno być tak:
```
public record TrainerDto(String name, String surname){
}
```

Reply:
I wtedy automatycznie dodaje gettery w formie `trainerDtoObject.name()`/`trainerDtoObject.surname()`